### DISCUSSION TITLE
Grandparent Rollup Full Recalc

### DISCUSSION THREAD

**Discussion by randybarton24** [jamessimone/apex-rollup#479] [Q&A]
@jamessimone sorry to continue pestering you... 🙈 I'm trying to run a full recalc on a grandparent rollup that I just configured.  It seems to run without any errors, but when I query the grandparent record to view results, it doesn't seem to be updating anything.  Do you see anything that I might be missing?
![image](https://github.com/jamessimone/apex-rollup/assets/8461377/51d5e58b-3c9c-4a68-b1e0-438d3c0ea19f)
![image](https://github.com/jamessimone/apex-rollup/assets/8461377/bfdbb73c-d635-4ea9-b0dc-b1b7e1efcd19)
![image](https://github.com/jamessimone/apex-rollup/assets/8461377/c1c7ab43-9de9-4599-aeb1-4805935780b8)

**Comment by jamessimone** [comment]
@randybarton24 never apologize! You're not bothering me. I've been able to fix a good number of really interesting bugs thanks to you. I'd have to get a look at one of the batch chunk logs to see what's going on, though. As far as setup goes, your grandparent paths look correctly formatted.
